lib/ohai/plugins/linux/platform.rb

```diff
@@ -44,16 +44,17 @@ def get_redhatish_platform(contents)
   # Amazon Linux 2
   # Amazon Linux 2 (Karoo)
   # Fedora release 28 (Twenty Eight)
+  # Fedora release 31 (Rawhide)
   # CentOS release 5.8 (Final)
   # CentOS release 6.7 (Final)
   # Red Hat Enterprise Linux Server release 7.5 (Maipo)
-  #
+  # Red Hat Enterprise Linux release 8.0 (Ootpa)
```

Comment:
You can replace all these examples with this:

# See https://rubular.com/r/78c1yXYa7zDhdV for example matches